test/performance/array/distCreate-large.ml-perf.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: blockDom:, blockArr:, cyclicArr:, cyclicDom:, blockCyclicDom:, blockCyclicArr:
+files: distCreate-large 
```

Comment:
`files` requires a list of files (that has the same length as `perfkeys`) If you want to use the same file for all `perfkeys` it's `repeat-files`